index.d.ts

```diff
@@ -404,3 +404,41 @@ export function stringifyUrl(
 	object: UrlObject,
 	options?: StringifyOptions
 ): string;
+
+/**
+Filter query parameters from a URL.
```

Comment:
This description about **removing** query parameters, while the below arguments is about **keeping**.

Reply:
By *filter*, I mean more like *filter out* except that wording might implicitly mean *take out* in this context. What do you think the description should be?